maintenance/multiversion.rst

```diff
@@ -183,8 +185,8 @@ By default we set it to FHIR R4 and R5, as for STU3 the fhirVersion may be unexp
 
 .. _feature_multi_version_r5:
 
-Support for R5 (experimental!)
-------------------------------
+Support for R5
+--------------
 
 By default the binaries for supporting R5 are included in the Firely Server distribution (since Firely Server (Vonk) 3.3.0). By default these binaries are not loaded. See the PipelineOptions in appsettings.default, where ``Vonk.Fhir.R5`` is commented out. Re-enable these in your appsettings.instance.
 
```

Comment:
Should we maybe add somewhere that errata for R5 are not yet included?

Reply:
It's right in the sentence below this sentence ;)